pbft/operations/pbft_persistent_operation.hpp

```diff
@@ -17,16 +17,17 @@
 #include <pbft/operations/pbft_operation.hpp>
 #include <storage/storage_base.hpp>
 #include <proto/pbft.pb.h>
+#include <peers_beacon/peers_beacon_base.hpp>
 
 namespace bzn
 {
     class pbft_persistent_operation : public pbft_operation
     {
     public:
-        pbft_persistent_operation(uint64_t view, uint64_t sequence, const bzn::hash_t& request_hash, std::shared_ptr<bzn::storage_base> storage, size_t );
+        pbft_persistent_operation(uint64_t view, uint64_t sequence, const bzn::hash_t& request_hash, std::shared_ptr<bzn::storage_base> storage, std::shared_ptr<bzn::peers_beacon_base> peers);
```

Comment:
Each operation is storing the peers beacon pointer. This seems a little redundant. Previously a peers pointer was stored but it could be different based on the config. In this case it will always be the same. 

It seems like it is only used in is_ready_for_(commit|execute), which are only called from pbft::maybe_advance_operation. Perhaps it could be passed in from there?

Reply:
It could be passed like that, yeah. The only reason I have written it like this is that it seemed more appropriate for the operation to know already, but that's a pretty vauge feeling.